Neos.Flow/Classes/Command/DoctrineCommandController.php

```diff
@@ -496,7 +496,7 @@ public function migrationGenerateCommand(bool $diffAgainstCurrent = true, string
             if ($selectedPackage !== $choices[0]) {
                 /** @var Package $selectedPackage */
                 $selectedPackage = $packages[$selectedPackage];
-                $targetPathAndFilename = Files::concatenatePaths([$selectedPackage->getPackagePath(), 'Migrations', $this->doctrineService->getDatabasePlatformName(), basename($migrationClassPathAndFilename)]);
+                $targetPathAndFilename = Files::concatenatePaths(['Packages', $selectedPackage->getPackagePath(), 'Migrations', $this->doctrineService->getDatabasePlatformName(), basename($migrationClassPathAndFilename)]);
```

Comment:
I don't think that the path was wrong here. Otherwise the next line would always create a wrong directory..
I guess that the bug is in line 502

Reply:
What @bwaidelich says!